src/HotChocolate/Spatial/src/Types.Spatial/CRS/GeoJsonCRSSchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using HotChocolate;
+
+namespace Types.Spatial.CRS
```

Comment:
- we should move extension methods on the `ISchemaBuilder` in the `HotChocolate` namespace so that people only need this namespace in their Startup.cs

- Extension methods on the `IServiceCollection` shall be in the `Microsoft.Extensions.DependencyInjection` namespace.